scripts/fast-reboot-dump.py

```diff
@@ -80,23 +77,45 @@ def get_bridge_port_id_2_port_id(db):
 
     return bridge_port_id_2_port_id
 
-def get_map_port_id_2_iface_name(db):
+def get_lag_by_member(member_name, app_db):
+    keys = app_db.keys(app_db.APPL_DB, 'LAG_MEMBER_TABLE:*')
+    keys = [] if keys is None else keys
+    for key in keys:
+        _, lag_name, lag_member_name = key.split(":")
+        if lag_member_name == member_name:
+            return lag_name
+    return None
+
+def get_map_port_id_2_iface_name(asic_db, app_db):
     port_id_2_iface = {}
-    keys = db.keys(db.ASIC_DB, 'ASIC_STATE:SAI_OBJECT_TYPE_HOSTIF:oid:*')
+    keys = asic_db.keys(asic_db.ASIC_DB, 'ASIC_STATE:SAI_OBJECT_TYPE_HOSTIF:oid:*')
     keys = [] if keys is None else keys
     for key in keys:
-        value = db.get_all(db.ASIC_DB, key)
+        value = asic_db.get_all(asic_db.ASIC_DB, key)
         if value['SAI_HOSTIF_ATTR_TYPE'] != 'SAI_HOSTIF_TYPE_NETDEV':
             continue
         port_id = value['SAI_HOSTIF_ATTR_OBJ_ID']
         iface_name = value['SAI_HOSTIF_ATTR_NAME']
         port_id_2_iface[port_id] = iface_name
 
+    keys = asic_db.keys(asic_db.ASIC_DB, 'ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER:oid:*')
```

Comment:
can we separate this logic in a method of its own? Something like `get_portchannel_by_member` will be more readable.

Reply:
The functionality of this part is the same as the one above only for LAG interfaces instead of HOST interfaces.
Both are adding elements to 'port_id_2_iface'.
You suggest to divide this two parts into two separate functions where each insert elements for different type of interface?

Reply:
Correct. It will be more readable

Reply:
@tahmed-dev I have refactored the code to be more readable, can you please review the change?